firestore/query.go

```diff
@@ -60,6 +60,13 @@ type Query struct {
 // in queries.
 const DocumentID = "__name__"
 
+// ArrayOperators contains the set of FieldFilter opterators that require an ArrayValue operand.
+var ArrayOperators = map[pb.StructuredQuery_FieldFilter_Operator]bool{
```

Comment:
Don't export a `var` unless it's okay for customers to overwrite the values. Either declare as `const` or hide (rename to camelCase).